web-components/src/components/modal/Modal.ts

```diff
@@ -271,8 +270,7 @@ export namespace Modal {
                   role="dialog"
                   id="${this.htmlId}"
                   class="md-modal ${classMap(this.modalContainerClassMap)}"
-                  aria-label="${this.ariaLabel}"
-                  aria-labelledby=${this.ariaLabelledBy}
+                  aria-label="${ifDefined(this.headerLabel || this.ariaLabel)}"
```

Comment:
Can you add undefined also if the both properties are not present then undefined will be activated and the aria-label will not be effective.
`aria-label="${ifDefined(this.headerLabel || this.ariaLabel || undefined)}"`

Reply:
done